extension.html

```diff
@@ -55,6 +55,44 @@ <h2>Introduction</h2>
 	The <code>getCoalescedEvents</code> API (similar to Android <a href="https://developer.android.com/reference/android/view/MotionEvent.html#getHistoricalX(int, int)">getHistorical</a> APIs and iOS <a href="https://developer.xamarin.com/api/member/MonoTouch.UIKit.UIEvent.GetCoalescedTouches/p/MonoTouch.UIKit.UITouch/">GetCoalescedTouches</a> API) exposes all the events that were coalesced into a single event. Coalescing events into a single event enables user agents to prevent processing of the events that don’t get a chance to update the frame content visually. Basically if the javascript handlers process the events and update the screen contents faster than refresh rate only the last update will be rendered and shown in the next frame. However, some applications like drawing apps may want to take advantage of the precise history of events as user moves the pointer on the screen to be able to draw smoother and better looking curves. This API can be used in those scenarios to get all the coalesced events from the event that was dispatched to the javascript.
       </p>
     </section>
+    <section>
+      <h2>Extensions to the Pointer Event types</h2>
+      <p>Below is a new pointer event type as well as a modification to an existing event in the <a href="https://w3c.github.io/pointerevents/index.html#pointer-event-types">existing pointer event types</a>.</p>
+      <section>
+        <h3>The <dfn><code>pointermove</code> event</dfn></h3>
+        <p>A user agent MUST <a href="#firing-events-using-the-pointerevent-interface">fire a pointer event</a> named <code>pointermove</code> when a pointer changes button state.
+        Additionally one <code>pointermove</code> MUST be fired when pointer changes coordinates, pressure, tangential pressure, tilt, twist, or
+        contact geometry (e.g. <code>width</code> and <code>height</code>) and the circumstances produce no other pointer events defined in this specification.
+        This <code>pointermove</code> that is not caused by button state change MUST be fired only once right before firing an event caused by button state change or
+        <a href="https://html.spec.whatwg.org/#event-loop-processing-model">animation frame callbacks</a>. If there are multiple changes to these attributes
```

Comment:
This doesn't seem to follow what implementations do. If there is nothing animating, *move events still fire at some point.
And I think events should fire sooner even in case there is something animating. More like before resize steps.

Reply:
What does this have to do with whether something being animated or not? Event if nothing is animating rAF is still there even if it doesn't run any callbacks in js. Can you explain what you tested? Chrome always does delay pointermoves with the current implementation to right before rAF.